recipes/roboticstoolbox-python/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "roboticstoolbox-python" %}
+{% set version = "0.9.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/roboticstoolbox-python-{{ version }}.tar.gz
+  sha256: efa9165125a07156097db5f6b6fc4990f7f8615d6a202c89f292a808da00aa26
+
+build:
+  number: 0
+  skip: true   # [py2k]
```

This looks like it can be `noarch: python`
```suggestion
  noarch: python
```